MP_10.11.12.14.15.16.17

nicolas-moya opened the pull request:
Se sube lo trabajado anteriormente respecto a
CRUD de producto
Ver inventario
Ver Catalago
En crear producto faltaria lo que es "Agregar categoria"

Commit:
MP_10.11.12.14.15.16.17

Se sube lo trabajado anteriormente respecto a
CRUD de producto
Ver inventario
Ver Catalago
En crear producto faltaria lo que es "Agregar categoria"

nicolas-moya commented:
perdon danilo xd

DaniloDaniel commented on controllers/ProductoController.php:
```diff
@@ -0,0 +1,154 @@
+    {
+        $model = new Producto();
+
+        /*if ($model->load(Yii::$app->request->post()) && $model->save()) {
```
Eliminar código comentado

DaniloDaniel commented on controllers/ProductoController.php:
```diff
@@ -0,0 +1,154 @@
+    {
+        $model = $this->findModel($id);
+
+        /*if ($model->load(Yii::$app->request->post()) && $model->save()) {
```
Eliminar código comentado.

DaniloDaniel commented on models/CategoriaSearch.php:
```diff
@@ -0,0 +1,68 @@
+
+        if (!$this->validate()) {
+            // uncomment the following line if you do not want to return any records when validation fails
+            // $query->where('0=1');
```
Eliminar código comentado.

DaniloDaniel commented on views/inventario/index.php:
```diff
@@ -0,0 +1,35 @@
+
+    <h1><?= Html::encode($this->title) ?></h1>
+
+    <?php // echo $this->render('_search', ['model' => $searchModel]); ?>
```
Eliminar código comentado.

DaniloDaniel commented on views/producto/_search.php:
```diff
@@ -0,0 +1,39 @@
+
+    <?= $form->field($model, 'activo') ?>
+
+    <?php // echo $form->field($model, 'categoria') ?>
```
Se usa?

DaniloDaniel commented on views/producto/_search.php:
```diff
@@ -0,0 +1,39 @@
+
+    <?php // echo $form->field($model, 'categoria') ?>
+
+    <?php // echo $form->field($model, 'imagen') ?>
```
Se usa?

Commit:
Actualizacion solicitada por Danilo

Se borraron los comentarios de los codigos solicitados

Commit:
MP_10.11.12.14.15.16.17_Update3

Se corrige el problema de catalogo al seleccionar el view de uno de estos

Se filtra catalogo para admin y para usuario
